gamemode/lang/portuguese.lua

```diff
@@ -1,76 +1,77 @@
 pt.bystander = "Inocente"
 pt.murderer = "Assassino"
-pt.spectator = "Observador"
+pt.spectator = "Espectador"
 
 pt.knife = "Faca"
 pt.magnum = "Magnum"
 pt.gun = "Arma"
 pt.hands = "Mãos"
 
 
-pt.teamSpectators = "Observadores"
+pt.teamSpectators = "Espectadores"
 pt.teamPlayers = "Jogadores"
 pt.teamAss = "Bolo"
 
-pt.killedTeamKill = "{player} matou um inocente!"
+pt.killedTeamKill = "{player} matou um inocente"
 pt.killedMurderer = "{player} matou o assassino"
 
-pt.murdererDeathUnknown = "O assassino morreu de circunstacias misteriosas"
+pt.murdererDeathUnknown = "O assassino morreu misteriosamente"
 
-pt.changeTeam = "{player} trocou de time para {team}"
+pt.changeTeam = "{player} entrou na equipe {team}"
 pt.teamMoved = "{player} foi movido para {team}"
-pt.teamMovedAFK = "{player} foi movido para {team} por ficar AFK"
+pt.teamMovedAFK = "{player} foi movido para {team} por estar ausente"
 
-pt.spectateFailed = "Você não pode observar. Você não está morto."
+pt.spectateFailed = "Você não pode espectar. Você não está morto."
 
-pt.murdererDisconnect = "O assassino não aguentou a pressão e saiu"
-pt.murdererDisconnectKnown = "O assassino saiu, ele era {murderer}"
+pt.murdererDisconnect = "O assassino deu rage quit"
```

Comment:
I think the old is better, "rage quit" it's not portuguese

Reply:
@ceifa Yes, "rage quit" isn't in Portuguese but a lot of people understand it. Like people also understand "first blood", "double kill", "triple kill", etc.